qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/update/UpdateCustomerDataSource.groovy

```diff
@@ -12,14 +11,6 @@ import life.qbic.portal.portlet.exceptions.DatabaseQueryException
  * @since: 1.0.0
  */
 interface UpdateCustomerDataSource {
-    /**
-     * This method returns a customer matching the given search criteria
-     *
-     * @param criteria a map with search criteria
-     * @return a person with affiliation and contact information
-     * @since 1.0.0
-     */
-    List<Customer> findCustomer(SearchCriteria criteria) throws DatabaseQueryException
```

Comment:
why did you delete it? Is there a replacement for this method?

Reply:
No but it is also no longer in any interface and I removed it to make the code simpler.